shared_queues_vendor/CMakeLists.txt

```diff
@@ -5,8 +5,9 @@ find_package(ament_cmake REQUIRED)
 
 include(ExternalProject)
 # Single producer single consumer queue by moodycamel - header only, don't build, install
-ExternalProject_Add(singleproducerconsumer
-  DOWNLOAD_DIR ${CMAKE_CURRENT_BINARY_DIR}/single
+ExternalProject_Add(ext-singleproducerconsumer
```

Comment:
I deliberately left in these changes because the cmake target of the `ExternalProject_Add` would collide with `add_library`.